packages/universal-crossplane/overlay/questions.yaml

```diff
@@ -0,0 +1,184 @@
+  show_subquestion_if: true
+  group: "Crossplane"
+  subquestions:
+  - variable: leaderElection
+    label: leaderElection
+    description: "Enable leader election for Crossplane Managers pod"
+    type: boolean
+    default: true
+    required: false
+    group: "Crossplane"
+  - variable: deploymentStrategy
+    label: deploymentStrategy
+    description: "The deployment strategy for the Crossplane and RBAC Manager (if enabled) pods"
+    type: enum
+    default: "RollingUpdate"
+    options:
+    - "RollingUpdate"
+    - "Recreate"
+    required: true
+    group: "Crossplane"
+  - variable: priorityClassName
+    label: priorityClassName
+    description: "Priority class name for Crossplane and RBAC Manager (if enabled) pods"
+    type: string
+    required: false
+    group: "Crossplane"
+  - variable: metrics.enabled
+    label: metrics.enabled
+    description: "Expose Crossplane and RBAC Manager metrics endpoint"
+    type: boolean
+    required: false
+    group: "Crossplane"
+# Basic Crossplane RBAC Manager configuration
+- variable: rbacManager.deploy
+  label: rbacManager.deploy
+  description: "Deploy RBAC Manager"
+  type: boolean
+  default: true
+  required: true
+  group: "Crossplane RBAC Manager"
+- variable: rbacManager.replicas
+  label: rbacManager.replicas
+  description: "The number of replicas to run for the RBAC Manager pods"
+  type: int
+  default: 1
+  required: true
+  group: "Crossplane RBAC Manager"
+# Advanced Crossplane RBAC Manager configuration
+- variable: advancedRBACManagerConfiguration
+  description: "Show advanced configuration for Crossplane RBAC Manager"
+  label: Show advanced configuration
+  type: boolean
+  default: false
+  show_subquestion_if: true
+  group: "Crossplane RBAC Manager"
+  subquestions:
+  - variable: rbacManager.leaderElection
+    label: rbacManager.leaderElection
+    description: "Enable leader election for RBAC Managers pod"
+    type: boolean
+    default: true
+    group: "Crossplane RBAC Manager"
+  - variable: rbacManager.managementPolicy
+    label: rbacManager.managementPolicy
+    description: "The extent to which the RBAC manager will manage permissions. All indicates to manage all Crossplane controller and user roles. Basic indicates to only manage Crossplane controller roles and the crossplane-admin, crossplane-edit, and crossplane-view user roles."
+    type: enum
+    default: "Basic"
+    options:
+    - "Basic"
+    - "All"
+    required: true
+    group: "Crossplane RBAC Manager"
+  - variable: rbacManager.skipAggregatedClusterRoles
+    label: rbacManager.skipAggregatedClusterRoles
+    description: "Opt out of deploying aggregated ClusterRoles"
+    type: boolean
+    default: true
+    group: "Crossplane RBAC Manager"
+# Basic Package configuration
+- variable: provider.packages
+  label: provider.packages
+  description: "The list of Provider packages to install together with Crossplane. Please use the 'Edit as YAML' for the best editing experience."
```

Comment:
Suggestion: `List of Provider packages to install with Crossplane. Select 'Edit as YAML' for the best editing experience.`